CHANGELOG.md

```diff
@@ -4,7 +4,11 @@
 *  Added desired time to DeliveryForm
 * GraphQL
     * Updated schema and resolver regarding desired time
-
+    * _Breaking_: New schema for products:
```

Comment:
please add **product** headline, maybe mention the changes in the product fake service

Reply:
done